src/components/Datagrid/Datagrid.tsx

```diff
@@ -0,0 +1,329 @@
+                  sticky: true,
+                }}
+                rightElement={
+                  <div
+                    className={classNames(classes.rowActionBar, {
+                      [classes.rowActionBarScrolledToRight]: scrolledToRight,
+                    })}
+                  >
+                    <div
+                      className={classNames(classes.rowActionBarShadow, {
+                        [classes.rowActionBarShadowActive]: !scrolledToRight,
+                      })}
+                    />
+                    <div className={classes.columnPicker}>
+                      <ColumnPicker
+                        IconButtonProps={{
+                          className: classes.columnPickerBtn,
+                          variant: "secondary",
+                          hoverOutline: false,
+                        }}
+                        availableColumns={availableColumnsChoices}
+                        initialColumns={columnChoices}
+                        defaultColumns={defaultColumns}
+                        onSave={onColumnsChange}
+                        hasMore={false}
+                        loading={false}
+                        onFetchMore={() => undefined}
+                        onQueryChange={picker.setQuery}
+                        query={picker.query}
+                      />
+                    </div>
+                    {hasColumnGroups && (
+                      <div
+                        className={classNames(classes.rowAction, {
+                          [classes.rowActionScrolledToRight]: scrolledToRight,
+                        })}
+                      />
+                    )}
+                    {Array(rowsTotal)
+                      .fill(0)
+                      .map((_, index) => (
+                        <div
+                          className={classNames(classes.rowAction, {
+                            [classes.rowActionSelected]: selection?.rows.hasIndex(
+                              index,
+                            ),
+                            [classes.rowActionScrolledToRight]: scrolledToRight,
+                          })}
+                          key={index}
+                        >
+                          <CardMenu
+                            disabled={index >= rowsTotal - added.length}
+                            Icon={MoreHorizontalIcon}
+                            IconButtonProps={{
+                              className: classes.columnPickerBtn,
+                              hoverOutline: false,
+                              state: "default",
+                            }}
+                            menuItems={menuItems(index)}
+                          />
+                        </div>
+                      ))}
+                  </div>
+                }
+                rowMarkerWidth={48}
+              />
+              {/* FIXME: https://github.com/glideapps/glide-data-grid/issues/505 */}
```

Comment:
In such a cases when we left `FIXME` or `TODO`, shouldn't we also make an issue in our dashboard repo and link it in the comment to keep track of it?

Reply:
will do, I have many things that I will create tasks for